webhooks/ssp_webhook.go

```diff
@@ -174,6 +183,23 @@ func validateDataImportCronTemplates(ssp *sspv1beta1.SSP) error {
 	return nil
 }
 
+func validateCommonInstancetypes(ssp *sspv1beta1.SSP) error {
+	if ssp.Spec.CommonInstancetypes == nil || ssp.Spec.CommonInstancetypes.URL == nil {
+		return nil
+	}
+
+	url := *ssp.Spec.CommonInstancetypes.URL
+	if strings.HasPrefix(url, "https://") && strings.HasPrefix(url, "ssh://") {
+		return fmt.Errorf("%s is invalid, only https:// or ssh:// are supported as a remote kustomize target for commonInstancetypes", url)
+	}
+
+	if !strings.Contains(url, "?ref=") {
```

Comment:
`version=` is valid too, see https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md.